lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -120,3 +124,132 @@ def get(self, request, username, course_id):
                 "grade": user_cert.get('grade')
             }
         )
+
+
+class CertificatesListView(GenericAPIView):
+    """
+        **Use Case**
+
+            * Get the list of viewable course certificates for a specific user.
+
+        **Example Request**
+
+            GET /api/certificates/v0/certificates/{username}
+
+        **GET Parameters**
+
+            A GET request must include the following parameters.
+
+            * username: A string representation of an user's username.
+
+        **GET Response Values**
+
+            If the request for information about the user's certificates is successful,
+            an HTTP 200 "OK" response is returned.
+
+            The HTTP 200 response contains a list of dicts with the following keys/values.
+
+            * username: A string representation of an user's username passed in the request.
+
+            * course_id: A string representation of a Course ID.
+
+            * course_display_name: A string representation of the Course name.
+
+            * course_organization: A string representation of the organization associated with the Course.
+
+            * certificate_type: A string representation of the certificate type.
+                Can be honor|verified|professional
+
+            * created_date: Date/time the certificate was created, in ISO-8661 format.
+
+            * status: A string representation of the certificate status.
+
+            * is_passing: True if the certificate has a passing status, False if not.
+
+            * download_url: A string representation of the certificate url.
+
+            * grade: A string representation of a float for the user's course grade.
+
+        **Example GET Response**
+
+            [{
+                "username": "bob",
+                "course_id": "edX/DemoX/Demo_Course",
+                "certificate_type": "verified",
+                "created_date": "2015-12-03T13:14:28+0000",
+                "status": "downloadable",
+                "is_passing": true,
+                "download_url": "http://www.example.com/cert.pdf",
+                "grade": "0.98"
+            }]
+    """
+
+    authentication_classes = (
+        JwtAuthentication,
+        OAuth2AuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+
+    permission_classes = (
+        C(IsAuthenticated) & (
+            C(permissions.NotJwtRestrictedApplication) & (
+                C(permissions.IsStaff) | permissions.IsUserInUrl
+            ) | (
+                C(permissions.JwtRestrictedApplication) &
+                permissions.JwtHasScope &
+                permissions.JwtHasUserFilterForRequestedUser
+            )
+        ),
+    )
+
+    required_scopes = ['certificates:read']
+
+    def get(self, request, username):
```

Comment:
Are you planning to add pagination?

Reply:
No. Is that something we need? If so, can you create a separate Jira ticket for that work?